src/main/java/org/benetech/servicenet/domain/OrganizationMatch.java

```diff
@@ -184,11 +101,8 @@ public int hashCode() {
     public String toString() {
```

Comment:
please remove toString since @ Data = @ getter, @ Setter, @ tostring, @ EqualsAndHashCode and @ RequiredArgsConstructor